plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/content/widget/RowContentListHelper.java

```diff
@@ -29,7 +29,11 @@
 		if (StringUtils.isBlank(contentsParameter)) {
 			return contents;
 		}
-		String parse1 = (contentsParameter.startsWith("[")) ? contentsParameter.substring(1) : contentsParameter;
+		String parse0 = contentsParameter
+				.replaceAll(", contentId=", ",contentId=")
+				.replaceAll(", modelId=", ",modelId=")
+				.replaceAll("\\}, \\{", "\\},\\{");
+		String parse1 = (parse0.startsWith("[")) ? parse0.substring(1) : parse0;
```

Comment:
Not the correct place to fix this. Please check `WidgetConfigPropertiesDeserializer`